packages/react-storage/src/components/StorageBrowser/composables/Search.tsx

```diff
@@ -18,9 +18,15 @@ const TOGGLE_BLOCK = 'toggle';
 
 export interface SearchProps {
   onSearch: (term: string, includeSubfolders: boolean) => void;
+  searchPlaceholder?: string;
+  showIncludeSubfolders?: boolean;
```

Comment:
Rather than add an prop that is specific to a singular `View` that will be surfaced on the composable API for both list View components, can we provide the toggle `Checkbox` to `Search` through composition?

Reply:
Yep 👍🏼